src/zkscript/fields/fq6_3_over_2.py

```diff
@@ -512,18 +570,25 @@ def mul_by_v(
         clean_constant: bool | None = None,
         is_constant_reused: bool | None = None,
     ) -> Script:
-        """Multiplication by v in F_q^6.
-
-        Input parameters:
-            - Stack: q .. X
-            - Altstack: []
-        Output:
-            - X * s
-        Assumption on data:
-            - X is passed as a triplet of elements of Fq2
-        Variables:
-            - If take_modulo is set to True, then the coordinates of the result are in Z_q; otherwise, the coordinates
-            are not taken modulo q.
+        """Multiplication by v in F_q^6 = F_q^2[v] / (v^3 - non_residue_over_fq2).
```

Comment:
is this function used in the codebase? (I think some of the multiplication for ... are not used anymore)

Reply:
Seems like it is being used in bls12_381/fields

Reply:
I would leave it anyway (regardless of whether it's used or not)